Parse/src/main/java/com/parse/ParseFileController.java

```diff
@@ -41,7 +41,7 @@ public ParseFileController(ParseHttpClient restClient, File cachePath) {
   /* package */ ParseHttpClient awsClient() {
     synchronized (lock) {
       if (awsClient == null) {
-        awsClient = ParsePlugins.get().newHttpClient();
+        awsClient = ParsePlugins.get().awsClient();
```

Comment:
Currently, the REST API uses a different HttpClient from the AWS client, and I was not really too sure of the reasons. Possibly just so that both could be making requests at the same time. Do you think they should use the same builder? I can make that the case, and they both just create their own client as needed if that sounds better.